dts/bindings/sensor/ti,bq274xx-common.yaml

```diff
@@ -40,3 +38,14 @@ properties:
         specific events happening (e.g. change in State of Charge). While in
         shutdown mode it acts as an input and toggling it will make the sensor
         exit the mode.
+
+    chemistry:
+      type: string
+      required: false
+      description: |
+        Chemistry type to use for devices that support programmable chemistry.
+        Currently this is only the BQ27426.
```

Comment:
This should be in the binding for that compatible, then, not the common file.

Reply:
done